services/integration.sh

```diff
@@ -78,15 +78,16 @@ function run() {
     instance=$2
     colour=$3
     # TODO can be removed once all services have been switched to YAML configs
-    export LOG_LEVEL=$4
+    # export LOG_LEVEL=$4
     ( ( cd "${DIR}/${service}" && "${TOP_LEVEL}/dist/${service}" -c "${service}${instance}.integration${integration_file_extension}" ) || kill_all) \
         | sed -e "s/^/$(tput setaf ${colour})[${service}] /" -e "s/$/$(tput sgr0)/" &
 }
 
 check_prerequisites
 
-run brig "" ${green}
-run galley "" ${yellow}
+# One still needs to pass a value since it's set in _run_
```

Comment:
then why is the setting in `integration.sh` commented out?

Reply:
This comment is not understandable outside of this PR change ("pass a value" - which value, where?), please rephrase.

Reply:
Because I forgot to uncomment, thanks!